gapic/src/main/com/google/gapid/perfetto/views/SliceSelectionView.java

```diff
@@ -77,6 +80,29 @@ public SliceSelectionView(Composite parent, State state, SliceTrack.Slice slice)
       createLabel(main, slice.name);
     }
 
+    RenderStageDebugMarkers debugMarkers = slice.getDebugMarkers();
+    if (debugMarkers != null) {
+      LinkedHashMap<String, String> props = new LinkedHashMap<String, String>();
+      if (debugMarkers.frameBufferHandle != 0 || !debugMarkers.frameBufferName.isEmpty()) {
```

Comment:
The handles should be shown whether or not the debug marker exist.

Reply:
Done